src/internal/nft-api-unichain.ts

```diff
@@ -0,0 +1,155 @@
+import { AlchemyConfig } from '../api/alchemy-config';
+import {
+  GetBaseNftsForOwnerUnichainOptions,
+  GetNftsForOwnerUnichainOptions,
+  Network,
+  OwnedBaseNftsResponse,
+  OwnedBaseNftsResponseUnichain,
+  OwnedNftsResponse,
+  OwnedNftsResponseUnichain
+} from '../types/types';
+import { getNftsForOwner } from './nft-api';
+import { NetworkPageKey, UnichainPageKeyCache } from './page-key';
+
+function networksAreEqual(a: Network[], b: Network[]): boolean {
+  return JSON.stringify(a.sort()) === JSON.stringify(b.sort());
```

Comment:
Better to not mutate the input. One way to do this would be to use `[...a].sort()`, etc. instead.